app/views/public_revisions/new_public_revision.html.haml

```diff
@@ -2,9 +2,9 @@
   - if @submission_id 
     = hidden_field_tag :submission_id, @submission_id 
   = f.input :status, collection: Submission::STATUS_MAPPING.
-  = f.input :file, label „File”, hint "Plik z nową wersją"
-  = f.input :pages, label „Number of pages”, hint „Ilość stron”
-  = f.input :pictures, label „Number of pictures”, hint „Ilość obrazków”
+  = f.input :file, label „Plik”, hint: "new"
```

Comment:
Teraz to zauważyłem - skąd Pani wzięła te cudzysłowy??? Co to za edytor?
Po `label` powinien być dwukropek.